ui/page/settingsStripeCard/view.jsx

```diff
@@ -419,7 +460,11 @@ class SettingsStripeCard extends React.Component<Props, State> {
         {currentFlowStage === 'confirmingCard' && (
           <div className="sr-root">
             <div className="sr-main">
-              <div className="sr-payment-form card cardInput">
+              <div className="">
+                <div className="sr-form-row">
+                  <label className="payment-details">Name On Card</label>
```

Comment:
i18n...

And would prefer `Name on card` for field labels, as that's what we've been trying to clean up everywhere, but it keeps getting back in.  

<img width="288" alt="image" src="https://user-images.githubusercontent.com/64950861/172978584-609fc275-b2b0-419f-afe0-b92a138c6c74.png">

Capitalized version will only be for section titles, and occasionally buttons (although lower case is still preferred).